files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md

```diff
@@ -268,14 +268,14 @@ ls | wc -l
 
 ## 一个稍微复杂一点的例子
 
-让我们看一些更复杂的东西。我们将首先尝试获取 MDN 的“获取”页面的内容 `curl` 命令 (可用于从 url 请求内容)[https://developer.mozilla.org/en-US/docs/Web/API/fetch](/zh-CN/docs/Web/API/fetch).
+让我们看一些更复杂的东西。我们将首先尝试获取 MDN 的“获取”页面的内容 `curl` 命令 (可用于从 url 请求内容)[https://developer.mozilla.org/zh-CN/docs/Web/API/fetch](/zh-CN/docs/Web/API/fetch).
 
 但是，这个 URL 是页面的旧位置。如果您在一个新的浏览器标签中输入它，您将 (最终) 被重定向到[https://developer.mozilla.org/enUS/docs/Web/API/fetch](/zh-CN/docs/Web/API/fetch).
```

Comment:
Not right here, I'd like to fix this in another PR.

![image](https://github.com/mdn/translated-content/assets/15844309/313e4fdc-2f4c-4afd-ae7b-a2cfd16b2d2b)